history.py

```diff
@@ -12,3 +12,19 @@
 channel.publish('play', 'bark')
 channel.publish('play', 'meow')
 channel.publish('play', 'cluck')
+
+print('Releasing channel...')
+# this part is only for tutorial purposes, in real you can use same instance of $channel
```

Comment:
We should remove this, but you are referencing the PHP variable $channel here anyway